packages/changed-elements-react/src/NamedVersionSelector/NamedVersionSelector.tsx

```diff
@@ -447,23 +454,28 @@ function NamedVersionSelectorLoaded(props: NamedVersionSelectorLoadedProps): Rea
   }
 
   return (
-    <List className="_cer_v1_named-version-list">
-      <Sticky className="_cer_v1_named-version-list-header">
-        <TextEx weight="semibold">
-          {t("VersionCompare:versionCompare.previousVersions")}
-        </TextEx>
-        {manageVersions}
-      </Sticky>
-      <namedVersionSelectorContext.Provider
-        value={{ processResults, viewResults, initialLoad, checkStatus }}
-      >
-        {
-          props.entries.map((entry) => (
-            <NamedVersionListEntry key={entry.namedVersion.id} entry={entry} />
-          ))
-        }
-      </namedVersionSelectorContext.Provider>
-    </List>
+    <>
+      <List className="_cer_v1_named-version-list">
+        <Sticky className="_cer_v1_named-version-list-header">
+          <TextEx weight="semibold">
+            {t("VersionCompare:versionCompare.previousVersions")}
+          </TextEx>
+          {manageVersions}
+        </Sticky>
+        <namedVersionSelectorContext.Provider
+          value={{ processResults, viewResults, initialLoad, checkStatus }}
+        >
+          {
+            props.entries.map((entry) => (
+              <NamedVersionListEntry key={entry.namedVersion.id} entry={entry} />
+            ))
+          }
+        </namedVersionSelectorContext.Provider>
+      </List>
+      <div>
+        {feedbackUrl ? <FeedbackButton feedbackUrl={feedbackUrl} /> : <></>}
```

Comment:
I'm rusty in frontend, so perhaps I'm missing something but: Is this always drawn with a bottom anchor point? Since you just used `<></>` I wonder if the named version list is short if the feedback button would move with it? It should always show at the bottom of the pane.

Also, I think you can simplify this with: `{feedbackUrl && <FeedbackButton ... />}`